src/locales/en/chapter4.js

```diff
@@ -420,7 +419,7 @@ export const lute = {
     "But if you play it with all your heart, it will open.", // lyla
     "I'm sure the person who made this mechanism also loved the goddess.", // lyla
     "That's nice.", // ann
-    "It's like the apple my daughter brought me.", // lyla
+    "It's like the apples you brought me.", // lyla // TODO: ask later
```

Comment:
I didn't understand how the sentence worked in with the rest of the conversation.

Reply:
Oh, I just realized that.
We maybe have different understanding of what the "it's like" is working for.

My "it's like" mentions goddess (or mind of "person who made this mechanism"). not for the apples.

So, it is
"It's (the goddess is) like the apples you brought me."
or
"It's (the person's mind is) like your mind when you collected these apples."

And next line may be
"It was delicious." // She already ate them at Line:400

Reply:
So is it "The goddess enjoys the apples your brought me."  ?  I'm still not quite following.

Reply:
Sorry, my explanation is difficult. :sweat_smile: 

Lyla wants to say Ann's kindness is similar to a mind of the person who made the goddess.
Because Ann brought only fresh apples.
(And I forgot to tell you that, If player picked some of damaged apples, Lyla doesn't say 422-425.)

Reply:
ohhhhhhhhhh I see.  These are difficult things to translate, I think I got it now!  "These apples are beautiful.  |The kindness you showed in picking them reminds me of the care the sculptor took when making the statue of this goddess."

Reply:
Thank you! I committed it.